flagset.go

```diff
@@ -224,9 +226,12 @@ func (f *FlagSetFiller) processStringToStringMap(fieldRef interface{}, hasDefaul
 		val = *casted
 	}
 	flagSet.Var(&strToStrMapVar{val: val}, renamed, usage)
+	if alias != "" {
+		flagSet.Var(&strToStrMapVar{val: val}, alias, usage)
+	}
```

Comment:
Assuming a switch to plural `aliases`:

```suggestion
	if aliases != "" {
		for _, alias := range strings.Split(aliases, ",") {
			flagSet.Var(&strSliceVar{ref: casted, override: override}, alias, usage)
		}
	}
```

Reply:
good addition :D
just updated the pr